transplant_guide.md

```diff
@@ -147,39 +147,67 @@ Info about U-district
+**Walking**
+  Don't be afraid to walk for transportation, if you live close enough. It might be a good idea to avoid heels until you get used to the hills and your feet adjust, but lots of Adies walk to class. Google Maps gives good estimates of how long it should take you to walk -- but don't forget to reverse the directions (e.g. look at how long it takes to walk _to_ Ada, and then hit reverse to see how long it takes to walk home _from_ Ada). The hills make the different directions take different times, and Google Maps accounts for this.
 
-Note: If you’re applying for the free Apple Care healthcare (which you can do at some of the ORCA LIFT enrollment offices), you’ll be able to skip this step since your lack of income will be verified then.
+**[ORCA LIFT](http://metro.kingcounty.gov/programs-projects/orca-lift/)**
+  So… you’re poor! Since it’s likely you’ll have zero income once attending Ada, it’s nice to know there are some perks with falling below the poverty line. The first of few is reduced bus fare! To apply for a reduced fare bus pass (the ORCA LIFT card), you’ll need to stop by one of these enrollment offices and provide proof of your eligibility and your identity.
+  We’ll warn you now: there are tons of locations, with tons of different hours. There's [one near Ada](http://www.kingcounty.gov/healthservices/health/locations/downtown.aspx) that's open 8am - 5pm. If you have your paperwork ready and show up right when they open (so there's little chance of a line), you can easily be out in 10-15 minutes.
+  They make you prove your income, and they have [tons of different options](http://metro.kingcounty.gov/programs-projects/orca-lift/to-qualify.html) for how to do that. Read carefully, and show up prepared. If you get Apple Health before you get the ORCA LIFT pass, they'll have it on file and you won't need to prove your income again.
+  Note: If you don't have any qualifying documentation to show your income, you can submit a request for your employment history (which can be done at the enrollment office). Expect it to take about a week, but you can ask the enrollment office to mail your ORCA LIFT card to your home so you don’t have to find another time to visit during their limited hours.
+  Note2: If you’re applying for the free Apple Care healthcare (which you can do at some of the ORCA LIFT enrollment offices), you’ll be able to skip this step since your lack of income will be verified then.
 
-- biking
-  - tips/tricks?
+**Bus Apps**
+  There are a lot of apps that can tell you which buses to take to get home. Options include:
+  - Google Maps
+  - SOMEONE ELSE NEEDS TO EXPAND THIS! I just use Google Maps :P
```

Comment:
I'll add more info to this one. I currently use OneBusAway & Google Maps. The bus system has their own app too called Trip Planner but it's not as good.